src/components/20-molecules/datepicker/story.js

```diff
@@ -20,7 +19,7 @@ storiesOf('Molecules/Datepicker', module)
     'Datepicker Input',
     () => `
       <axa-datepicker inputfield year="2019" month="0" day="1" data-test-id="datepicker-input-2019"></axa-datepicker>
-      <axa-datepicker inputfield year="2020" month="1" day="2" data-test-id="datepicker-input-2020"></axa-datepicker>
-      <axa-datepicker inputfield year="2021" month="2" day="3" data-test-id="datepicker-input-2021"></axa-datepicker>
+      <axa-datepicker inputfield placeholder="Bitte Datum wählen" year="2020" month="1" day="2" data-test-id="datepicker-input-2020" invaliddatetext="Ungültiges Datum"></axa-datepicker>
```

Comment:
We should keep the documenation in english, including all values.

Reply:
Definitely a no from me here. We should use realistic values from the domain we develop for, which is AXA.